index.html

```diff
@@ -206,6 +206,42 @@ <h1 class="abt-title">About me</h1>
           </ul>
         </div>
       </section>
+      <section class="contact-section">
+        <div class="contact-text">
+          <h4>I'm always interested in hearing about new projects,
+             so if you'd like to chat please get in touch.</h4>
+        </div>
+        <div class="contact-form">
+          <form action="https://formspree.io/f/mzbywpkb" method="POST">
+            <input type="text" id="name" name="user_name" placeholder="Full name" minlength="1" maxlength="30" required />
+            <input type="email" id="mail" name="email_address" placeholder="Email" required />  
+            <textarea id="comment" placeholder="Enter text here" name="comment" minlength="1" maxlength="500" required ></textarea>
+            <button type="submit" name="submit" id="submit">Get in touch</button>
+          </form>
```

Comment:
- [ ] Hello. The code is written correctly but the problem is that the contact form is not displayed on the live demo of your project. Please kindly fix that.

Reply:
Already fixed!